Pod/Classes/ios/NYTPhotoViewController.m

```diff
@@ -157,8 +157,8 @@ - (void)didDoubleTapWithGestureRecognizer:(UITapGestureRecognizer *)recognizer {
     CGPoint pointInView = [recognizer locationInView:self.scalingImageView.imageView];
     
     CGFloat newZoomScale = self.scalingImageView.maximumZoomScale;
-    
-    if (self.scalingImageView.zoomScale >= self.scalingImageView.maximumZoomScale) {
+
+    if ((self.scalingImageView.zoomScale - self.scalingImageView.maximumZoomScale) >= 0.0) {
```

Comment:
That's a good point, but I don't think we have any test coverage for the `UIKit` stuff in general so that becomes a much bigger task